cpp/src/Ice/ConnectRequestHandler.cpp

```diff
@@ -89,29 +89,6 @@ ConnectRequestHandler::getConnection()
     return nullptr;
 }
 
-Ice::ConnectionIPtr
-ConnectRequestHandler::waitForConnection()
```

Comment:
waitForConnection was dead code: we declared and implemented it in a number of classes but never called it.